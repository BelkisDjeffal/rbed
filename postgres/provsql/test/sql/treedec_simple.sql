\set ECHO none
SET search_path TO provsql_test,provsql;

CREATE TABLE treedec_simple_result AS
SELECT *, probability_evaluate(provenance(),'tree-decomposition') AS prob FROM personnel
ORDER BY id;

SELECT remove_provenance('treedec_simple_result');

SELECT id, ROUND(prob::numeric,1) AS prob FROM treedec_simple_result;
DROP TABLE treedec_simple_result;
