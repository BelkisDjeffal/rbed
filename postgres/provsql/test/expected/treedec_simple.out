\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

 id | prob 
----+------
  1 |  0.1
  2 |  0.2
  3 |  0.3
  4 |  0.4
  5 |  0.5
  6 |  0.6
  7 |  0.7
(7 rows)

